stories/Components/Forms/Inputfields/Input_group.jsx

```diff
@@ -2,29 +2,48 @@ import React from 'react';
 import './input_group.scss';
 
 export const Inputcomponent = ({
-  id, type, element, rows, cols, placeholder, required, mode, labelText, errorText, disabled, minlength, helpText,
+  id, type, element, rows, cols, placeholder, required, labelText, errorText, minlength, helpText, State,
 }) => {
   const InputTag = `${element}`;
+  
+  let state;
+  switch(State){
```

Comment:
@Shamimkhan321 As per @kuldev feedback we can use the following here:
```
let state;
state = State ? State.toLowerCase() : 'default';
```
or if you need to check it against certain set of states. You can also write it as:
```
let state;
const states = ["Focus", "Error", "Disabled"];
state = states.includes(State) ? State.toLowerCase() : 'default';
```